fms_core/resources/_utils.py

```diff
@@ -28,3 +29,11 @@ def skip_rows(dataset: Dataset, num_rows: int = 0, col_skip: int = 1) -> None:
         if len(vals) == 1 and "" in vals:
             continue
         dataset.append(tuple(str_normalize(c) if isinstance(c, str) else ("" if c is None else c) for c in r))
+
+
+def remove_column_from_results(column_name: str, results):
+    index_column = results.diff_headers.index(column_name)
+    results.diff_headers.remove(column_name)
+    for row in results.rows:
+        row.diff.pop(index_column)
```

Comment:
If I submit a sample update template with values in both `New Volume` and `Delta Volume` I get an error: `'NoneType' object has no attribute 'pop'`. Probably because `row.diff` is None in this case.